src/OmniSharp/project.json

```diff
@@ -12,6 +12,7 @@
         "Microsoft.Framework.Logging.Console": "1.0.0-beta3",
         "Microsoft.Framework.ConfigurationModel.Json": "1.0.0-beta3",
         "Microsoft.Framework.Cache.Memory": "1.0.0-beta3",
+        "Microsoft.Framework.FileSystemGlobbing": "1.0.0-*",
         "Microsoft.Composition": "1.0.30"
```

Comment:
@davidfowl - this resolves to a beta4 library which from understanding conflicts with the changes in framework naming. In effect, OmniSharp gives me a compile error for ```aspnetcore50``` that it cannot find ```Matcher```.

![image](https://cloud.githubusercontent.com/assets/1794099/7159341/c1e44f82-e381-11e4-8eb3-4544a6043162.png)